backend/main-service/src/main/java/kkakka/mainservice/cart/application/CartService.java

```diff
@@ -1,7 +1,48 @@
 package kkakka.mainservice.cart.application;
 
+import kkakka.mainservice.cart.domain.Cart;
+import kkakka.mainservice.cart.domain.repository.CartItemRepository;
+import kkakka.mainservice.cart.domain.repository.CartRepository;
+import kkakka.mainservice.cart.ui.dto.CartRequestDto;
+import kkakka.mainservice.product.domain.Product;
+import kkakka.mainservice.product.domain.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.Optional;
+
 @Service
 public class CartService {
+
+    CartRepository cartRepository;
+    CartItemRepository cartItemRepository;
+    ProductRepository productRepository;
+
+    @Autowired
+    public CartService(CartRepository cartRepository, CartItemRepository cartItemRepository, ProductRepository productRepository) {
+        this.cartRepository = cartRepository;
+        this.cartItemRepository = cartItemRepository;
+        this.productRepository = productRepository;
+    }
```

Comment:
```suggestion
@Service
public class CartService {

    private final CartRepository cartRepository;
    private final CartItemRepository cartItemRepository;
    private final ProductRepository productRepository;

    public CartService(CartRepository cartRepository, CartItemRepository cartItemRepository, ProductRepository productRepository) {
        this.cartRepository = cartRepository;
        this.cartItemRepository = cartItemRepository;
        this.productRepository = productRepository;
    }
```

`@Autowired` 는 빼도 될거예요
이왕 생성자 주입 방식으로 DI를 해주고 있으니 `private final`로 선언해 생성자 주입 방식의 이점을 누려봅시당!

Reply:
생성자주입 방식의 이점을 누려보겠습니다 !